sdcm/cluster.py

```diff
@@ -1604,6 +1604,21 @@ def remote_manager_yaml(self):
     def remote_manager_agent_yaml(self):
         return self._remote_yaml(path=SCYLLA_MANAGER_AGENT_YAML_PATH)
 
+    def get_db_node_instance_type(self):
```

Comment:
May be create it as a property? and add type hints?

Reply:
Good idea. I added it as a property on the BaseNode.